### PR TITLE
Upgrade debezium to 2.3.0.Final

### PR DESCRIPTION
**Pull request by tzolov**

### CODE REVIEW

File: applications/source/debezium-source/README.adoc [hunk 1]
```diff
@@ -104,32 +104,32 @@ Those properties can be used by prefixing them by the `debezium.properties.` pre
...
 
-|https://debezium.io/documentation/reference/2.2/connectors/vitess.html[Vitess]
-|https://debezium.io/documentation/reference/2.2/connectors/vitess.html#vitess-connector-properties
+|https://debezium.io/documentation/reference/2.3/connectors/vitess.html[Vitess]
+|https://debezium.io/documentation/reference/2.3/connectors/vitess.html#vitess-connector-properties
 
-|https://debezium.io/documentation/reference/2.2/connectors/spanner.html[Spanner]
-|https://debezium.io/documentation/reference/2.2/connectors/spanner.html#spanner-connector-properties
+|https://debezium.io/documentation/reference/2.3/connectors/spanner.html[Spanner]
+|https://debezium.io/documentation/reference/2.3/connectors/spanner.html#spanner-connector-properties
```
Comment (artembilan) [inline applications/source/debezium-source/README.adoc:132]: That's exactly what I meant about support burden if we don't use "versionless" links which essentially points to the latest versions anyway.
Like this one: https://debezium.io/documentation/reference/connectors/spanner.html#spanner-connector-properties

🤷

File: functions/supplier/debezium-supplier/pom.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
     <description>Debezium Supplier</description>
 
     <properties>
-        <version.debezium>2.3.0.CR1</version.debezium>
+        <version.debezium>2.3.0.Final</version.debezium>
```
Comment (artembilan) [inline functions/supplier/debezium-supplier/pom.xml:18]: I believe we can manage this version in a single place where we would just import `debezium-bom`

**Review by artembilan — CHANGES_REQUESTED** [review]

File: functions/supplier/debezium-supplier/pom.xml (hunk 2 above)
Comment (tzolov) [reply functions/supplier/debezium-supplier/pom.xml:18]: Yeh, this is a recurring discussion. 
Currently the functions don't have a bom for external dependencies.  The existing `function-dependecies` is only providing a BOM for sharing internal dependencies between the `functions` and `applications`. 
So should we start crating a bom project for evry function common project that might need it or create one shared e.g. something like `function-common-dependecies` bom? 
For either decision is it right to do it at this release stage?

Comment (artembilan) [reply functions/supplier/debezium-supplier/pom.xml:18]: Heads up! 
Turned out it is wrong to use `debezium-bom`.
They have too many not project related deps over there.
The point of BOM to manage only your own modules, but not the rest of transitive deps.
🤷

**Review by artembilan — CHANGES_REQUESTED** [review]

**Pull request merged**